requirements_dev.txt

```diff
@@ -1,14 +1,15 @@
-pip==19.1.1
+pip==19.3.1
 bumpversion==0.5.3
 watchdog==0.9.0
-flake8==3.7.7
-tox==3.9.0
-coverage==4.5.3
-Sphinx==2.0.1
+flake8==3.7.8
+tox==3.14.0
+coverage==4.5.4
+sphinx==2.2.0
 sphinx-rtd-theme==0.4.3
 
-twine==1.13.0
+twine==1.15.0
```

Comment:
I would keep twine 1.x and add a comment that twine 2.0 doesn't support py3.5.

Otherwise:

There seems to be a way to specify environment-specific dependencies in requirements files, which could be an option (but I would do this if twine 2 had particular features that we need)
https://www.python.org/dev/peps/pep-0508/#environment-markers